packages/e2e/nightwatch/config.js

```diff
@@ -1,15 +1,23 @@
 const config = {
   domain: "http://localhost:3000", // @TODO read from environment variable
   selectors: {
+    conclusionContact: "[data-testid=need-contact]",
+    conclusionNeedPermitButton: "[data-testid=need-permit-button]",
+    introBullets: "[data-testid=intro-usable-for-bullets]",
```

Comment:
Maybe out of scope for this PR, but it would be really nice if the selectors are in ie. `e2e/selectors.js`.
This way we can reuse them for puppeteer also.

Reply:
Good point. Fixed this